package.json

```diff
@@ -13,72 +13,79 @@
+    "build-examples-doc": "mkdirp projects/examples/gen && node dist/ng-live-docs/index.js projects/examples/tsconfig.app.json projects/examples/src/main.ts projects/examples/gen/examples-doc.json",
     "generate-translations": "node projects/components/src/assets/index.js && copyfiles -f projects/components/src/assets/i18n.json dist/components",
     "generate-example-translations": "node projects/i18n/scripts/index.js dist/components/i18n.json projects/examples/src/assets/our-translations.json projects/examples/src/assets/i18n.json"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "10.2.3",
-    "@angular/common": "10.2.3",
-    "@angular/compiler": "10.2.3",
-    "@angular/core": "10.2.3",
-    "@angular/forms": "10.2.3",
-    "@angular/platform-browser": "10.2.3",
-    "@angular/platform-browser-dynamic": "10.2.3",
-    "@angular/router": "10.2.3",
-    "@clr/angular": "4.0.10",
-    "@clr/core": "4.0.10",
-    "@clr/icons": "4.0.10",
-    "@clr/ui": "4.0.10",
+    "@angular-devkit/architect": "^0.1202.16",
+    "@angular/animations": "12.2.16",
+    "@angular/common": "12.2.16",
+    "@angular/compiler": "12.2.16",
+    "@angular/core": "12.2.16",
+    "@angular/forms": "12.2.16",
+    "@angular/platform-browser": "12.2.16",
+    "@angular/platform-browser-dynamic": "12.2.16",
+    "@angular/router": "12.2.16",
+    "@cds/core": "^5.6.4",
+    "@clr/angular": "12.0.12",
+    "@clr/icons": "5.0.1",
+    "@clr/ui": "12.0.12",
+    "@messageformat/core": "^3.0.1",
     "@ngrx/store": "10.1.2",
     "@ngx-formly/core": "5.6.1",
-    "@stackblitz/sdk": "1.3.0",
+    "@stackblitz/sdk": "^1.3.0",
     "@types/yargs": "17.0.5",
     "@vcd/angular-client": "0.0.2-alpha.1",
     "@vcd/bindings": "9.1.1",
-    "@vmw/ng-live-docs": "0.0.13",
-    "@vmw/plain-js-live-docs": "0.0.5",
+    "@webcomponents/custom-elements": "^1.5.0",
     "@webcomponents/webcomponentsjs": "2.0.0",
     "@wessberg/ts-evaluator": "0.0.25",
-    "angular-cli-ghpages": "0.6.2",
-    "classlist.js": "1.1.20150312",
+    "angular-cli-ghpages": "1.0.0",
     "codecov.io": "0.1.6",
     "copyfiles": "^2.4.1",
-    "karma-coverage": "2.0.2",
+    "cross-spawn": "^7.0.3",
+    "karma-coverage": "~2.0.3",
     "lint-staged": "^12.3.7",
-    "lit-html": "1.1.2",
     "messageformat": "1.0.2",
     "mkdirp": "^1.0.4",
     "mousetrap": "1.6.5",
-    "prismjs": "1.27.0",
+    "postcss": "^8.4.4",
+    "prismjs": "^1.23.0",
     "properties": "1.2.1",
-    "rbradford-compodoc": "1.1.11",
+    "rbradford-compodoc": "^1.1.11",
     "rxjs": "6.5.5",
+    "schematics-utilities": "^2.0.2",
     "systemjs": "0.20.19",
-    "tsickle": "0.39.1",
-    "tslib": "2.0.0",
+    "tsickle": "^0.39.1",
```

Comment:
Remove the ^ in the package versions?

Reply:
We have decide to use loose versions for smaller node_modules and fewer dependency version problems